### PR TITLE
[Feature:RainbowGrades] Border outline reflects status

### PR DESCRIPTION
**Pull request by ziesski**
fixes Submitty/Submitty#9112
fixes Submitty/Submitty#9331 (EDIT (06.27.2023) - from student.cpp line 474 implementing LookupGradeable etc. 

Rainbow Grade will now have border outline for 4 status.
Black border outline = Academic integrity violation
Yellow border outline = Grade override
Green border outline = Grade inquiry in progress
Read border outline = Bad status
 

Works for both individual & instructor (transposed) version
Screen Shot 2023-06-02 at 12 53 42 PM
Screen Shot 2023-06-02 at 12 59 50 PM

### CODE REVIEW

File: output.cpp [hunk 1]
```diff
@@ -973,6 +973,7 @@ void start_table_output( bool /*for_instructor*/,
       color = GradeColor(g);
       if (this_student->getMossPenalty() < -0.01) {
         g += "@";
+          std::cout << this_student->getUserName() << std::endl;
```
Comment (bmcutler) [inline output.cpp:976]: remove debug print statement

File: student.cpp [hunk 2]
```diff
@@ -466,6 +491,7 @@ void Student::mossify(const std::string &gradeable, float penalty) {
   moss_penalty += -0.0000001;
   moss_penalty += -average_letter_grade * penalty;
 
+    std::cout << "Moss penalty: student.cpp 481 " << moss_penalty << std::endl;
```
Comment (bmcutler) [inline student.cpp:494]: remove debug print statement

**Review by bmcutler — CHANGES_REQUESTED** [review]
tested it, good improvement
just remove a couple excess debugging print statements and lets merge it.

File: output.cpp (hunk 1 above)
Comment (ziesski) [reply output.cpp]: fixed

File: student.cpp (hunk 2 above)
Comment (ziesski) [reply student.cpp]: fixed

**Pull request merged**